build_image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# A simple script to build the Docker images.
+# This is intended to be invoked as a step in Argo to build the docker image.
+#
+# build_image.sh ${DOCKERFILE} ${IMAGE} ${TAG}
+set -ex
+
+DOCKERFILE=$1
+CONTEXT_DIR=$(dirname "$DOCKERFILE")
+IMAGE=$2
+TAG=$3
+
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+cd $CONTEXT_DIR
+
+echo "Building ${IMAGE} using gcloud build"
+gcloud builds submit --tag=${IMAGE}:${TAG} --project=${GCP_PROJECT} .
```

Comment:
Can you also look at how tag is created?  https://github.com/kubeflow/pytorch-operator/blob/master/scripts/build.sh#L38 
This will ensure that unique tag is created for every presubmit

Reply:
With suggested change, docker version will get changed for every submission(new docker will be created for every submission in both presubmit and postsubmit) ? Is that ok?

Reply:
Isn't that you need? You need to have a unique image tag created for every presubmit. And you will separately add a new tag for new release. eg: v0.7 tag for upcoming release

Reply:
I was thinking, manually we can update docker version only when operatror code changes instead of changing for every submission.
Automatic unique tag generation is always fine(except container registry size issue), I will resubmit as suggested.
Thanks for your comment.

Reply:
@johnugeorge 
One info, currently xgboost-operator repo doesn't have any tags hence docker image tag name will be having some unique tag but it may not be in proper format like pytorch or anyother images.

till we create some branch, we may have to bear with this tags.

Reply:
newly built image is `gcr.io/kubeflow-ci/xgboost-operator:de89cf4`